docs/data/date-pickers/timezone/timezone.md

```diff
@@ -0,0 +1,161 @@
+|          `"locale"` | Will use the system's local timezone                                                                                                                                                                                                                                                                                                                                                                           |
+| IANA standard zones | Example: `"Europe/Paris"`, `"America/New_York"`<br/>[List of all the IANA zones](https://timezonedb.com/time-zones)                                                                                                                                                                                                                                                                                            |
+|        Fixed offset | Example: `"UTC+7"`<br/>**Only available with Luxon**                                                                                                                                                                                                                                                                                                                                                           |
+
+## Usage with Day.js
+
+[Dayjs timezone documentation](https://day.js.org/docs/en/timezone/timezone)
+
+### Day.js and UTC
+
+To be able to use UTC, you have to enable the `utc` plugin:
+
+```tsx
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+
+dayjs.extend(utc);
+```
+
+{{"demo": "DayjsUTC.js"}}
+
+### Day.js and timezones
+
+To be able to use timezones, you have to enable both the `utc` and the `timezone` plugin:
+
+```tsx
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+```
+
+{{"demo": "DayjsTimezone.js"}}
+
+## Usage with Luxon
+
+[Luxon timezone documentation](https://moment.github.io/luxon/#/zones)
+
+### Luxon and UTC
+
+{{"demo": "LuxonUTC.js"}}
+
+### Luxon and timezone
+
+{{"demo": "LuxonTimezone.js"}}
+
+## Usage with Moment
+
+[Moment timezone documentation](https://momentjs.com/timezone/)
+
+### Moment and UTC
+
+{{"demo": "MomentUTC.js"}}
+
+### Moment and timezone
+
+To be able to use timezones, you have to pass the default export from `moment-timezone` to the `dateLibInstance` prop of `LocalizationProvider`:
+
+```tsx
+import moment from 'moment-timezone';
+import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+
+export default function App({ children }) {
+  return (
+    <LocalizationProvider dateAdapter={AdapterMoment} dateLibInstance={moment}>
+      {children}
+    </LocalizationProvider>
+  );
+}
+```
+
+{{"demo": "MomentTimezone.js"}}
+
+## Props with another timezone
```

Comment:
Might be better at the top of the page (just before stating library specific sections) because 

- It answers one of the question I asked myself while reading how to use TZ in DayJS. 
- Reader might stop at the section about the library they use and assume the remaining page is about other libraries. And so they might never discover this section (very sad).

If moved upper in the page structure, it could end with a sentence highlighting the existence of "Common examples" section. For example

> For more advanced timezone manipulation, please refer to the Common examples section

Reply:
I removed this section, now that we distinguish the rendering timezone and the value timezone.